src/job/Job.ts

```diff
@@ -8,7 +8,7 @@ export type MomoJobStatus = WithoutId<JobEntity>;
 export interface JobDefinition {
   name: string;
   interval: string;
```

Comment:
would it make sense to also pass interval as a number (in milliseconds), analogous to firstRunAfter?

Reply:
Yeah, it seems weird that the interval can only be given as human-readable string.